.travis.yml

```diff
@@ -5,50 +5,55 @@ addons:
   apt:
     packages:
     - socat
-    - enchant
-    - aspell
-    - aspell-en
 
 env:
   global:
-  - REDIS_TAGS="2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1" INSTALL_DIR=$HOME/redis
+  - REDIS_TAGS="2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1" INSTALL_DIR=$HOME/redis TEST_ARGS="-n 4"
```

Comment:
Better `-n auto`?

Reply:
Probably... 
Once I run tests with `-n auto` and it started 32 processes, now when I was adding this change it run with only 2 processes...

Reply:
AFAIR pytest-xdist is intelligent enough to identify Travis CI and set this to 2 processes. If it can access the corresponding env vars, of course...

Reply:
Oh, and there's `PYTEST_ADDOPTS` special var for this purpose...

P.S. I like mapping-style var definition more, it's better readable.